clang/lib/Headers/avx512fintrin.h

```diff
@@ -43,6 +43,7 @@ typedef unsigned short __mmask16;
 
 /* Rounding mode macros.  */
 #define _MM_FROUND_TO_NEAREST_INT   0x00
+#define _MM_FROUND_TIES_TO_EVEN     0x00
```

Comment:
(very pedantic) but _MM_FROUND_TO_NEAREST_TIES_EVEN would more closely keep to the existing `_MM_FROUND_TO_*` naming scheme